skimage/transform/_thin_plate_splines.py

```diff
@@ -93,25 +99,26 @@ def estimate(self, src, dst):
         #     raise ValueError("src and dst must have shape (N,2)")
 
         self.control_points = src
-        n , d = src.shape
+        n, d = src.shape
 
-        K = self._radial_distance(src)
+        K = self._radial_distance(src) + np.eye(n, dtype=np.float32)
```

Comment:
Can you explain why you are using ones for the diagonal instead of zeros? I think the original implementation and the paper expect the diagonal of K to be 0.

Reply:
An idea I got from a paper published in 2003 titled 'Approximation Methods for Thin Plate Spline Mappings and Principal Warps'. There, they multiplied the value of K with a unit matrix and positive scalar term `λ`(optional). 
Just to see its affects the results of the final output. Will revert to the original paper.